RELEASE_CHECKLIST.md

```diff
@@ -13,29 +13,23 @@ If you are releasing a beta or a release candidate, no official changelog is nee
    milestones.
 1. Go through the commit history since the last release. Ensure that all PRs
    that have landed are marked with the milestone. You can use this to
-   show all the PRs that are merged on or after YYY-MM-DD:
-   `https://github.com/issues?utf8=%E2%9C%93&q=repo%3Aapollographql%2Fapollo-cli+merged%3A%3E%3DYYYY-MM-DD`
+   show all the PRs that are merged on or after YYYY-MM-DD:
+   `https://github.com/issues?utf8=%E2%9C%93&q=repo%3Aapollographql%2Frover+merged%3A%3E%3DYYYY-MM-DD`
 1. Go through the closed PRs in the milestone. Each should have a changelog
    label indicating if the change is documentation, feature, fix, or maintenance. If
    there is a missing label, please add one. If it is a breaking change, also add a BREAKING label.
 1. Add this release to the `CHANGELOG.md`. Use the structure of previous
    entries.
 
-### Update the README with the latest help strings
-
-1. Run `cargo run -- help` and copy the output into the README.
-
-### Update cargo manifest
-
-1. Update the version in `Cargo.toml`.
-1. Run `cargo update`.
-1. Run `cargo test --workspace`.
-1. Make sure you have `npm` installed, and run `cargo build`.
-
 ### Start a release PR
 
+1. Make sure you have both `npm` and `cargo` installed on your machine and in your `PATH`.
 1. Create a new branch "#.#.#" where "#.#.#" is this release's version (release) or "#.#.#-rc.#" (release candidate)
-1. Push up a commit with the `Cargo.toml`, `Cargo.lock`, `CHANGELOG.md`, and `./installers/npm` changes. The commit message can just be "#.#.#" (release) or "#.#.#-rc.#" (release candidate)
+1. Update the version in `Cargo.toml`.
+1. Run `cargo xtask prep` (this will require `npm` to be installed).
```

Comment:
It would be super good in the future to have this run as part of `release` task in CI. Works as a manual step for now though!

Reply:
Yeah - the only thing here is that I like to have all of this prep stuff committed in the tag. One thing we could do is have the job on like a [repository dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#repository_dispatch).

ideally you'd run something like `cargo xtask release v0.2.0` that would:

1) change the version in `Cargo.toml` to `v0.2.0`
1) run `cargo xtask prep`
1) commit the auto generated changes
1) tag the changes
1) push the tag

this would then trigger our regular release workflow that runs on pushed tags.

I think for now it's fine but that's sorta my idea for further automating this

Reply:
yea I think that'd be a really good flow, and we can do it at a later stage!